util/StringTable.cpp

```diff
@@ -57,7 +57,7 @@ void StringTable::Load(std::shared_ptr<const StringTable> fallback) {
     auto path = FilenameToPath(m_filename);
     std::string file_contents;
 
-    bool read_success = parse::read_file(path, file_contents);
+    bool read_success = ReadFOCScript(path, file_contents);
```

Comment:
A stringtable isn't FOCS

Reply:
Aren't it parsed for FOCS substitutions?

Reply:
Macro substitutions aren't [FO] content scripts. Even if they were, this function doesn't do them.

Reply:
OK, I removed FOCS mention here. Moreover I suddenly fixed code duplication in `ShaderProgram`.